### PR TITLE
Data Table Conversions: Part one

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16571

*Note:*
- To alleviate QA and code review, will create a follow up ticket for the remaining tables, as the changes required there are substantial. 
- Hide white space for easier reviewing 👍 

*Description of changes:*
* Converts the follow v-data-tables to v-simple-tables
- [x] CertifyInformation
- [x] Vehicle Collateral
~- [ ] HomeOwnersTable~ - Follow up ticket
- [x] Debtors
- [x] RegisteringParty
- [x] SecuredParties
- [x] BasePartySummary
~- [ ] RegistrationTable~ - Follow up ticket
- [x] HomeSectionsTable
~- [ ] SearchedResults (ppr)~ - Follow up ticket
- [x] SearchedResults (mhr)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1371-06quj2nh.web.app

File: ppr-ui/src/components/parties/party/RegisteringParty.vue [hunk 1]
```diff
@@ -83,42 +87,46 @@
...
-              <v-btn
-                id="retry-registering-party"
-                outlined
-                color="primary"
-                @click="getRegisteringParty()"
-              >
-                Retry <v-icon>mdi-refresh</v-icon>
-              </v-btn>
-              <error-contact class="search-contact-container pt-6" />
+                    </div>
```
Comment (orelbn) [inline ppr-ui/src/components/parties/party/RegisteringParty.vue:87]: Can you double check the spacing here please

File: ppr-ui/src/components/parties/party/SecuredParties.vue [hunk 2]
```diff
@@ -108,249 +108,264 @@
...
+            <tbody v-if="securedParties.length > 0">
+              <tr
+                v-for="(item, index) in securedParties"
+                :key="item.partyId"
+                class="party-row"
+                :class="{ 'disabled-text-not-action': item.action === ActionTypes.REMOVED}"
+              >
+                <!-- Edit Form -->
+                <template v-if="showEditParty[index]">
+                  <tr v-if="showEditParty[index]">
```
Comment (orelbn) [inline ppr-ui/src/components/parties/party/SecuredParties.vue]: Is the inner `v-if` necessary here?

**Review by orelbn — APPROVED** [review]
Looks good a couple small comments.

File: ppr-ui/src/components/parties/party/RegisteringParty.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/parties/party/RegisteringParty.vue:87]: Definitely, thank you!

File: ppr-ui/src/components/parties/party/SecuredParties.vue (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/parties/party/SecuredParties.vue]: Good eye, probably not! Will have a look

**Review by dimak1 — APPROVED** [review]
Good job, nicely done. Just some conflicts to resolve.

**Comment by cameron-eyds** [comment]
> Good job, nicely done. Just some conflicts to resolve.

Thank you!

File: ppr-ui/src/components/parties/party/RegisteringParty.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/parties/party/RegisteringParty.vue:87]: Cleaned up!

File: ppr-ui/src/components/parties/party/SecuredParties.vue (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/parties/party/SecuredParties.vue]: Fixed!

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1371-06quj2nh.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1371-06quj2nh.web.app

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Pull request merged**